src/install/index.js

```diff
@@ -250,27 +271,22 @@ const deploySteps = (apps, mode, deployDoc, firstRun, ddoc) => {
     .then(getApplicationDdoc)
     .then(ddoc => {
       return stage('horti.stage.deploying', 'Deploying new installation')
-        .then(() => performDeploy(apps, mode, deployDoc, ddoc, firstRun))
-        .then(() => stage('horti.stage.postCleanup', 'Post-deploy cleanup, installation complete'))
-        .then(() => postCleanup(deployDoc));
+        .then(() => performDeploy(mode, deployDoc, ddoc, firstRun))
+        .then(() => {
+          stage('horti.stage.postCleanup', 'Post-deploy cleanup, installation complete');
+          return postCleanup(ddocWrapper(ddoc, mode), deployDoc);
+        });
     });
 };
 
 
 
 module.exports = {
-  // TODO: when all is said and done
-  //       do we still need apps, and first run?
-  //       (cause you can intuit them?)
-  //  (
-  //    you know what apps exist because they are in the application ddoc list
-  //    you know if its first run because the apps are either running or they're not
-  //  )
```

Comment:
Did you answer this question fully? We are still passing around `firstRun`, so it still looks to have some relevance